cmd/vela-slack/plugin.go

```diff
@@ -187,6 +187,13 @@ func (p *Plugin) Exec() error {
 	return nil
 }
 
+func cleanBuildMessage(buildMessage string) string {
+	subject := buildMessage
+	subject = strings.Replace(subject, "\n", "\\n", -1)
+	subject = strings.Replace(subject, "\"", "\\\"", -1)
```

Comment:
**question:** N.b. I didn't look for some kind of a library function that would make a string "safe to inject into JSON". I speculate that `json.Marshal` might do this safely.

Reply:
minor: could you update to use `.ReplaceAll()` instead